server/src/main/java/org/cloudfoundry/identity/uaa/resources/jdbc/AbstractQueryable.java

```diff
@@ -43,6 +44,10 @@ public void setQueryConverter(SearchQueryConverter queryConverter) {
         this.queryConverter = queryConverter;
     }
 
+    public void setNamedParameterJdbcTemplate(NamedParameterJdbcTemplate namedParameterJdbcTemplate) {
```

Comment:
this can be removed / refactored, if https://github.com/cloudfoundry/uaa/pull/2864 is merged before